scripts/collect_ticks.py

```
@@ -0,0 +1,296 @@
+            continue
+        cid = info["conditionId"]
+        if cid not in windows:
+            windows[cid] = {
+                "series": series_slug, "slug": info["slug"],
+                "start_ts": info["start_ts"], "end_ts": info["end_ts"],
+                "up_token": info["up_token"], "down_token": info["down_token"],
+                "seen_tape": set(), "snap_count": 0,
+                "label": label, "duration": duration,
+            }
+        w = windows[cid]
+        time.sleep(random.uniform(0.0, JITTER_SEC))
```

Ruff (0.16.2)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)